py21cmsense/data/profiles/MWA-PhaseII.yaml

```diff
@@ -0,0 +1,15 @@
+antpos: "../mwa_phase_compact_antpos.txt"
```

Comment:
is there a reason this one is a file and the HERA is a quantity?

Reply:
just to illustrate the different ways to do it?

Reply:
woops. fixed.